### PR TITLE
Fetch External API

### PR DESCRIPTION
**Pull request by markobrkovic**
- Install the `node-fetch` dependency to fetch external API from the server side more easily
- Add the Backend fetch
- Add one Frontend fetch for mostly testing purposes
- Install the `cors` dependency and use it to fix the CORS issue

### CODE REVIEW

File: server/index.ts [hunk 1]
```diff
@@ -1,16 +1,20 @@
...
 
-if (!process.env.MONGODB_URI) {
+if (!process.env.MONGODB_URI || !process.env.CLIENT_ID || !process.env.AT) {
   throw new Error('NO MONGODB URL dotenv variable');
 }
 
 const port = process.env.PORT || 3001;
 const app = express();
+app.use(cors({ origin: '*' }));
+
```
Comment (phlgr) [inline server/index.ts:17]: Cors should not be a problem in the backend.

File: server/index.ts [hunk 2]
```diff
@@ -42,6 +48,34 @@ app.post('/api/games', async (request, response) => {
   }
 });
 
+// Fetching from twitch API
+
+app.get('/api/twitchgames', async (_req, res) => {
+  console.log('/twitchgames endpoint called');
+  const options = {
+    'Client-ID': `${process.env.CLIENT_ID}`,
+    Authorization: `Bearer ${process.env.AT}`,
```
Comment (phlgr) [inline server/index.ts:57]:
```suggestion
    Authorization: `Bearer ${process.env.TOKEN}`,
```
Please no abbreviations

File: server/tsconfig.json [hunk 3]
```diff
@@ -20,5 +20,5 @@
     "module": "commonjs",
     "noEmit": false
   },
-  "include": ["*.ts"]
+  "include": ["*.ts", "index.ts"]
```
Comment (phlgr) [inline server/tsconfig.json:23]:
```suggestion
  "include": ["*.ts"]
```
This already includes all `.ts` files. Try restarting the TS Server or your VSCode

File: src/App.tsx [hunk 4]
```diff
@@ -16,6 +16,8 @@ function App() {
     //   .then((result) => setMessage(result.message));
   }, []);
 
+  //localhost:3001/api/twitchgames
+
```
Comment (phlgr) [inline src/App.tsx:20]: Please no comments

File: src/utils/fetchAPI.ts [hunk 5]
```diff
@@ -0,0 +1,6 @@
+export default async function games() {
+  const response = await fetch('http://localhost:3001/api/twitchgames');
+  const body = await response.json();
+  alert(body.results);
```
Comment (phlgr) [inline src/utils/fetchAPI.ts:4]:
```suggestion
```

File: src/utils/fetchAPI.ts [hunk 6]
```diff
@@ -0,0 +1,6 @@
+export default async function games() {
```
Comment (phlgr) [inline src/utils/fetchAPI.ts:1]:
```suggestion
export default async function getGames() {
```
Try to describe what the function actually does

**Review by phlgr — CHANGES_REQUESTED** [review]

File: src/utils/fetchAPI.ts (hunk 6 above)
Comment (markobrkovic) [reply src/utils/fetchAPI.ts]: Damn it, I forgot to push my last commit, I already did that and other suggested stuff, thanks though :(

**Pull request merged**